torch/testing/_internal/common_methods_invocations.py

```diff
@@ -20957,6 +20952,26 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
         "_refs.vsplit",
         torch_opinfo_name="vsplit",
     ),
+    PythonRefInfo(
+        "_refs.dot",
+        torch_opinfo_name="dot",
+        # .conj() returns a view on complex tensors??
+        validate_view_consistency=False,
```

Comment:
Why aren't we getting the view behavior right here?

Reply:
At a guess, `return torch.dot(self, other.conj()).conj()` probably needs to come outside of `elementwise_type_promotion_wrapper`.

Reply:
Not quite. I checked under this branch:
```python
    elif other.is_conj():
        return torch.dot(self, other.conj()).conj()
```
which is the one that triggers the issue, and `torch.dot(self, other.conj())` is not a view, but `torch.dot(self, other.conj()).conj()` is a view.
On the other hand, if I put the same code in ATen, it says that `torch.dot(self, other.conj()).conj()` is conj but not a view... I'm submitting a different fix.

Reply:
Nvm, the other fix doesn't work. I'm not quite sure what's going on there.
I'm voting for leaving as is right now as we don't even support complex numbers at the moment in inductor.

Reply:
Okay the eager result having `is_conj()` but not `_is_view()` is weird but not really our fault here, so I guess thats fine. This comment should be updated to reflect that though; `.conj()` is expected to return a view so the comment doesn't make much sense.